golang/cosmos/x/vpurse/types/codec.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+var (
+	amino = codec.NewLegacyAmino()
```

Comment:
ew.

Reply:
Yeah.  There's nothing about amino that isn't yucky.